src/ImageSharp/PixelFormats/PixelImplementations/A8.cs

```diff
@@ -55,6 +56,12 @@ public partial struct A8 : IPixel<A8>, IPackedVector<byte>
     [MethodImpl(InliningOptions.ShortMethod)]
     public static bool operator !=(A8 left, A8 right) => !left.Equals(right);
 
+    /// <summary>
+    /// Gets the The pixel type information.
+    /// </summary>
+    /// <returns>PixelTypeInfo</returns>
```

Comment:
You should use `/// <inheritdoc/>` for each implementation.

Reply:
I first did that. But SA1648 is giving a false positive here. Should I disable it globally or in each file?